### PR TITLE
Uses model translation in order adjustments table

### PR DESCRIPTION
**Pull request by Murph33**
Improve I18n use by using translations off of a model instead of arbitrary ones.  Eases localization.

Cherry picked from #549 and is part of an ongoing effort to improve I18n use as discussed in #735.

### CODE REVIEW

**Comment by cbrunsdon** [comment]
:+1:

File: backend/app/views/spree/admin/orders/_adjustments.html.erb [hunk 1]
```diff
@@ -3,8 +3,8 @@
     <legend><%= title %></legend>
     <table>
       <thead>
-        <th><%= Spree.t('name')%></th>
-        <th><%= Spree.t('amount')%></th>
+        <th><%= Spree::Adjustment.human_attribute_name(:name)%></th>
```
Comment (jhawthorn) [inline backend/app/views/spree/admin/orders/_adjustments.html.erb]: I think this should be `:label`, as that is the attribute we are printing

Comment (Murph33) [inline backend/app/views/spree/admin/orders/_adjustments.html.erb]: `:label` is already translated to "Description" though from #743.  If we want to maintain the same naming scheme from having the column as `name` we'd either need some kind of `:label_name` thing in the dictionary or to redo the other PR or change the appearance of this page

Comment (tvdeyen) [inline backend/app/views/spree/admin/orders/_adjustments.html.erb]: Exactly. I didn't want to change the original appearance of the page. We can change that later on, as `label` would be the correct attribute.

Comment (jhawthorn) [inline backend/app/views/spree/admin/orders/_adjustments.html.erb]: Sure

**Pull request merged**